plugins/CoreHome/javascripts/popover.js

```diff
@@ -142,7 +142,17 @@ var Piwik_Popover = (function () {
 
         /** Set the title of the popover */
         setTitle: function (titleHtml) {
+            var titleText = $('<div>' + titleHtml + '</div>').text();
+            if (titleText.length > 60) {
+                titleHtml = $('<span>').attr('class', 'tooltip').attr('title', titleText).html(titleHtml);
+            }
             container.dialog('option', 'title', titleHtml);
+            try {
+                $('.tooltip', container.parentNode).tooltip('destroy');
+            } catch (e) {}
+            if (titleText.length > 60) {
```

Comment:
Is it maybe possible to just add a title attribute to the title after it was created? I reckon this one creates the tooltip with black background etc?